geocity/apps/submissions/payments/models.py

```diff
@@ -116,3 +119,186 @@ def get_refund_pdf(self, read=False):
+        on_delete=models.CASCADE,
+        verbose_name=_("Demande"),
+        related_name="submissioncfc2price",
+        help_text=_("Demande"),
+    )
+    cfc2_price = MoneyField(
+        default=0.0,
+        decimal_places=2,
+        max_digits=12,
+        default_currency="CHF",
+        verbose_name=_("Montant CFC 2"),
+    )
+
+
+class ServicesFeesType(models.Model):
+    administrative_entity = models.ForeignKey(
+        AdministrativeEntity,
+        null=True,
+        on_delete=models.CASCADE,
+        verbose_name=_("Entité administrative"),
+        related_name="administrative_entity",
+        help_text=_("Entité administrative."),
+    )
+    name = models.CharField(
+        verbose_name=_("Prestation"),
+        max_length=255,
+        null=False,
+    )
+    fix_price = MoneyField(
+        default=None,
+        null=True,  # For hourly service fees, this field has to be null
+        decimal_places=2,
+        max_digits=12,
+        default_currency="settings.DEFAULT_CURRENCY",
+        verbose_name=_("Tarif forfaitaire [CHF]"),
+        help_text=_("Le tarif forfaitaire de cette prestation."),
+    )
+    is_visible_by_validator = models.BooleanField(
+        verbose_name=_("Visible par le validateur"),
+        help_text=_("Est visible par le validateur"),
+    )
+    integrator = models.ForeignKey(
+        Group,
+        null=True,
+        on_delete=models.SET_NULL,
+        verbose_name=_("Groupe des administrateurs"),
+        limit_choices_to={"permit_department__is_integrator_admin": True},
+    )
+
+    class Meta:
+        verbose_name = _("2.4 Type de prestation")
+        verbose_name_plural = _("2.4 Types de prestation")
+
+    # Methods
+    def __str__(self):
+        return f"{self.name}"
+
+
+class ServicesFees(models.Model):
+    """Docstring"""
+
+    # Hidden yet mandatory fields
+    # created_* and updated_* fields to keep tracks of the user that has
+    # effectively created or updated the current prestation.
+    # Those fields SHOULD NOT be exposed in the form.
+    # Use the other ones, e.g. provided_*
+    created_at = models.DateTimeField(
+        verbose_name=_("Date de création."),
+        auto_now_add=True,
+    )
+    updated_at = models.DateTimeField(
+        verbose_name=_("Date de dernière modification."),
+        auto_now=True,
+    )
+    created_by = models.ForeignKey(
+        User,
+        null=True,
+        on_delete=models.SET_NULL,
+        max_length=255,
+        verbose_name=_("Créée par"),
+        related_name="service_fee_created_by",
+        help_text=_("La prestation a été créé par cet utilisateur."),
+    )
+    updated_by = models.ForeignKey(
+        User,
+        null=True,
+        on_delete=models.SET_NULL,
+        max_length=255,
+        verbose_name=_("Mise à jour par"),
+        related_name="service_fee_updated_by",
+        help_text=_("La prestation a été mise à jour par cet utilisateur."),
+    )
+    permit_department = models.ForeignKey(
```

Comment:
Can you not find the departement from the user groups and the submission entity ?